### PR TITLE
Case creation smoketest

### PR DESCRIPTION
**Pull request by rachelschneiderman**
https://trello.com/c/cTu4IWca/566-case-creation

### CODE REVIEW

File: cypress-smoketests/fixtures/example.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Using fixtures to represent data",
+  "email": "hello@cypress.io",
+  "body": "Fixtures are a great way to mock data for responses to routes"
+}
```
Comment (rachaelparris) [inline cypress-smoketests/fixtures/example.json:5]: Does this file need to be included?

**Review by rachaelparris — COMMENTED** [review]
Have these been tested on a deployed environment yet?

**Comment by rachelschneiderman** [comment]
> Have these been tested on a deployed environment yet?

Yes! We have run these tests multiple times on dev.

**Review by rachaelparris — APPROVED** [review]

File: cypress-smoketests/fixtures/example.json (hunk 1 above)
Comment (rachelschneiderman) [reply cypress-smoketests/fixtures/example.json:5]: I don't believe so, I will remove it in the next PR.

**Pull request merged**